### PR TITLE
WIP release notes

### PR DESCRIPTION
**Pull request by sspencerwire**
* this is just the template (copy of 9.4) for now
* updated `.pages` file included

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/release_notes/9_5.md [hunk 1]
```diff
@@ -146,7 +124,7 @@ Many known issues exist in 9.5 including those related to:
 * Supportability
 * Containers
 
-Review the [upstream listing](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/known-issues) for the complete details on these issues.
+Review the [upstream listing](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#__WKANCHOR_5g) for the complete details on these issues.
```
Comment (grayeul) [inline docs/release_notes/9_5.md:127]: The link to RedHat Known Issues is wrong.  It should be WKANCHOR_5i  (not 5g)

File: docs/release_notes/9_5.md [hunk 2]
```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
...
-* New processor microarchitectures supported in `papi`
-* New package `maven-openjdk21`
-* New package `libzip-tools`
-* `cmake` rebased to version 3.26
+* **GCC Toolset 14**
+* **LLVM Toolset 18.1.8**
+* **Rust Toolset 1.79.0**
+* **Go Toolset 1.22**
 
 ## Known issues
```
Comment (grayeul) [inline docs/release_notes/9_5.md:107]: I suggest adding a specific callout to the Installer Known Issues, referencing the blank display help:

* In the installer, the Help pages are not displayed properly. The text is there for copy/paste but is invisible

File: docs/release_notes/9_5.md [hunk 3]
```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
...
-### Image builder
-
-* From Rocky Linux 9.4 forward, you can specify arbitrary custom mount points except for specific paths that are reserved for the operating system
-* Creation of different partitioning modes is now available including `auto-lvm`, `lvm`, and `raw`
-* Tailor options for a profile and add them to your blueprint customizations. You can accomplish this by selecting and unselecting options to add and remove rules.
-
 ### Security
 
-Listed below are security-related highlights in the latest Rocky Linux 9.4 release. For a complete list of security-related changes, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/new-features#new-features-security).
+Listed below are security-related highlights in the latest Rocky Linux 9.5 release. For a complete list of security-related changes, see the [upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23new-features-security).
```
Comment (grayeul) [inline docs/release_notes/9_5.md:61]: Move to HTML link: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.5_release_notes/new-features#new-features-security

File: docs/release_notes/9_5.md [hunk 4]
```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
...
+* **SELinux** The SELinux policy now provides a boolean that allows QEMU Guest Agent to execute confined commands 
+* **OpenSSL** TLS toolkit upgrades to 3.2.2. OpenSSL now supports certificate compression and adds in Brainpool curves to the TLS 1.3 protocol.
+* **ca-certificates** program now provides trusted CA roots in the OpenSSL directory format format
+* **crypto-policies** packages are updated to extend its control to algorithm sections in Java
+* **NSS** cryptographic toolkit are rebased to upstream version 3.101
 
 ### Dynamic programming languages, web, and database servers
 
-For a detailed list of the changes in this category see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/new-features#new-features-dynamic-programming-languages-web-and-database-servers).
+For a detailed list of the changes in this category see the [upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23technology-previews-dynamic-programming-languages-web-and-database-servers).
```
Comment (grayeul) [inline docs/release_notes/9_5.md:71]: Move to HTML: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.5_release_notes/new-features#new-features-dynamic-programming-languages-web-and-database-servers

File: docs/release_notes/9_5.md [hunk 5]
```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
...
-* **nginx 1.24** available as a module stream
-* **MariaDB 10.11** available as a module stream
-* **PostgreSQL 16** available as a module stream
+* **Apache HTTP Server 2.4.62**
+* **Node.js 22**
+
+### Compilers and development tools
 
-The following applications have been upgraded:
+For details on the changes in this category [see the upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23new-features-compilers-and-development-tools)
```
Comment (grayeul) [inline docs/release_notes/9_5.md:80]: HTML: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.5_release_notes/new-features#new-features-compilers-and-development-tools

**Review by grayeul — COMMENTED** [review]
Added comments for changing PDF links to HTML

File: docs/release_notes/9_5.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+* Graphics infrastructure
+* Virtualization
+* Supportability
+* Containers
+
+Review the [upstream listing](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#__WKANCHOR_5g) for the complete details on these issues.
+
+!!! info "Deprecated Software"
+
+    Some Application Streams, which are part of the Rocky Linux 9.4 repositories, won't receive any further updates. These currently include Node.js 16 (retirement April 2024) and .NET 7 (retirement May 2024). Furthermore OpenJDK 11 will reach it's EoL for Rocky Linux 9 in October 2024. Please make sure to use supported versions (Node.js 18 or 20 modules, .NET 6 or 8 packages, OpenJDK 1.8.0, 17 or 21 packages) if further patches are desired.
```
Comment (lumarel) [inline docs/release_notes/9_5.md:131]:
```suggestion
    Some Application Streams, which are part of the Rocky Linux 9.5 repositories, won't receive any further updates. These currently include gcc-toolset 12 (retirement Nov 2024), .NET 6 (retirement Nov 2024), Ruby 3.1 (retirement March 2025) and Node.js 18 (retirement April 2025). Furthermore OpenJDK 11 will reach it's EoL for Rocky Linux 9 in October 2024. Please make sure to use supported versions (gcc-toolset 13 or 14 packages, .NET 8 or 9 packages, the Ruby 3.3 module, Node.js 20 or 22 modules, OpenJDK 1.8.0, 17 or 21 packages) if further patches are desired.
```

**Comment by lumarel** [comment]
Lol, same changes ✋🏻 @grayeul

File: docs/release_notes/9_5.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+* Kernel
+* File systems and storage
+* Dynamic programing languages, web and database servers
+* Identity management
+* Desktop
+* Graphics infrastructure
+* Virtualization
+* Supportability
+* Containers
+
```
Comment (lumarel) [inline docs/release_notes/9_5.md:126]:
```suggestion

### Help in Anaconda

Currently the help buttons when clicked in Anaconda only show blank pages.

```

A suggestion for @grayeul 's comment.

File: docs/release_notes/9_5.md (hunk 1 above)
Comment (lumarel) [reply docs/release_notes/9_5.md:127]: God you covered 👍🏻

File: docs/release_notes/9_5.md (hunk 2 above)
Comment (lumarel) [reply docs/release_notes/9_5.md:107]: And here too 👍🏻

File: docs/release_notes/9_5.md (hunk 5 above)
Comment (lumarel) [reply docs/release_notes/9_5.md:80]: Think you took the wrong line but also added this 👍🏻

**Review by lumarel — APPROVED** [review]

File: docs/release_notes/9_5.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+* **Rust Toolset 1.79.0**
+* **Go Toolset 1.22**
+
+Updated default Java implementation to `java-17-openjdk`:  
+After this update the `java`, `java-devel` and `jre` packages will install Java 17 instead of Java 11.
+
+### Web console
+
+The `cockpit-files` package got added for easier file management in the web console.
+This file manager can perform basic tasks as browsing the files and directories, copy/move/delete and download/upload files.
```
Comment (SergeCroise) [inline docs/release_notes/9_5.md:105]:
```suggestion
This file manager can perform basic tasks such as browsing the files and directories, copy/move/delete and download/upload files.
```

File: docs/release_notes/9_5.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+As with any release, there are many known issues. Review the [upstream listing](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/9.5_release_notes/index#known-issues-installer-and-image-creation) for the complete details on these issues.
+
+### Help in Anaconda
+
+Currently the help buttons when clicked in Anaconda only show blank pages.
+
+!!! info "Deprecated Software"
+
+    Some Application Streams, which are part of the Rocky Linux 9.5 repositories, won't receive any further updates. These currently include gcc-toolset 12 (retirement Nov 2024), .NET 6 (retirement Nov 2024), Ruby 3.1 (retirement March 2025) and Node.js 18 (retirement April 2025). Furthermore OpenJDK 11 will reach it's EoL for Rocky Linux 9 in October 2024. Please make sure to use supported versions (gcc-toolset 13 or 14 packages, .NET 8 or 9 packages, the Ruby 3.3 module, Node.js 20 or 22 modules, OpenJDK 1.8.0, 17 or 21 packages) if further patches are desired.
```
Comment (SergeCroise) [inline docs/release_notes/9_5.md:127]:
```suggestion
    Some Application Streams, which are part of the Rocky Linux 9.5 repositories, won't receive any further updates. These currently include gcc-toolset 12 (retirement Nov 2024), .NET 6 (retirement Nov 2024), Ruby 3.1 (retirement March 2025) and Node.js 18 (retirement April 2025). Furthermore OpenJDK 11 will reach its EoL for Rocky Linux 9 in October 2024. Please make sure to use supported versions (gcc-toolset 13 or 14 packages, .NET 8 or 9 packages, the Ruby 3.3 module, Node.js 20 or 22 modules, OpenJDK 1.8.0, 17 or 21 packages) if further patches are desired.
```

**Pull request merged**